CounterStrike2GSI/Nodes/Provider.cs

```diff
@@ -27,12 +27,18 @@ public class Provider : Node
         /// </summary>
         public readonly string SteamID;
 
+        /// <summary>
+        ///  Timestamp of the GameState data.
+        /// </summary>
+        public readonly long Timestamp;
+
         internal Provider(JObject parsed_data = null) : base(parsed_data)
         {
             Name = GetString("name");
             AppID = GetInt("appid");
             Version = GetInt("version");
             SteamID = GetString("steamid");
+            Timestamp = GetLong("timestamp");
```

Comment:
Disassembly shows that the function used for populating `appid` and `version` fields is the same used for `timestamp`.

```suggestion
            Timestamp = GetInt("timestamp");
```